From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 21 Sep 2015 20:06:55 +0200
Subject: Use daemon log facility for journald

otherwise messages end up in /var/log/kern.log if journald forwards to
syslog.

Closes: #799633
---
 src/util/virlog.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/util/virlog.c b/src/util/virlog.c
index b45ee91..627f4cb 100644
--- a/src/util/virlog.c
+++ b/src/util/virlog.c
@@ -920,6 +920,7 @@ virLogOutputToJournald(virLogSourcePtr source,
     journalAddString(&state, "MESSAGE", rawstr);
     journalAddInt(&state, "PRIORITY",
                   virLogPrioritySyslog(priority));
+    journalAddInt(&state, "SYSLOG_FACILITY", LOG_DAEMON);
     journalAddString(&state, "LIBVIRT_SOURCE", source->name);
     if (filename)
         journalAddString(&state, "CODE_FILE", filename);
